.gitignore

```diff
@@ -2,4 +2,8 @@ bin/
 obj/
 /packages/
 riderModule.iml
-/_ReSharper.Caches/
\ No newline at end of file
+/_ReSharper.Caches/
+BackendTests/BackendTests.csproj
+Backend.sln.DotSettings.user
```

Comment:
Should the file be removed in this PR?

Reply:
I didn't quite understand what you meant in the PR description.

Reply:
Sorry, when I read it back it wasn't too clear. I just meant that when we've merged this one, we should probably remove all IDE/user-specific files so when we branch out again, the repo is more clean.

I tried removing it now just so I didn't further dirty the file. However the ignore did not work, as the file is still there.

Reply:
I see, makes sense.
I'll create an issue.

Reply:
See #28.